src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/SimplifyBooleanExpressionCheck.java

```diff
@@ -39,6 +39,31 @@
  * <pre>
  * &lt;module name=&quot;SimplifyBooleanExpression&quot;/&gt;
  * </pre>
+ * <p>Example: </p>
+ * <pre>
+ * public class Test {
+ *
+ *  public void bar(){
+ *
+ *   boolean a, b;
+ *   Foo c, d, e;
+ *
+ *   if(!false){}; // violation, can be simplified to true
+ *
+ *   if(a == true){}; // violation, can be simplified to a
+ *   if(a == b){}; // OK
+ *   if(a == false){}; // violation, can be simplified to !a
+ *   if(!(a != true)){}; // violation, can be simplified to a
+ *
+ *   e = (a || b) ? c : d; // OK
+ *   e = (a || false) ? c : d; // violation, can be simplified to a
+ *   e = (a &amp;&amp; false) ? c : d; // violation, can be simplified to false
```

Comment:
`can be simplified to false`
This example looks weird, because it can be simplified to just `e = d`

Reply:
Got it.